sklearn/__init__.py

```diff
@@ -15,6 +15,9 @@
 import sys
 import logging
 import os
+import random
+
+import numpy as np
```

Comment:
Let's keep `import numpy as np` in `setup_module`, because we want to be able to import from `sklearn` during setup when numpy is not installed.
```suggestion
```

Reply:
Thanks for the suggestion @thomasjpfan ! Made that change and pushed!